db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
```

Comment:
Does it matter that it doesn't reference another table but another record in this table? Probably not, I just have a very rudimentary understanding of indexes